lms/djangoapps/instructor/tests/views/test_instructor_dashboard.py

```diff
@@ -228,7 +228,7 @@ def test_dashboard_analytics_points_at_insights(self):
         Test analytics dashboard message is shown
         """
         response = self.client.get(self.url)
-        analytics_section = '<li class="nav-item"><button type="button" class="btn-link" data-section="instructor_analytics">Analytics</button></li>'  # pylint: disable=line-too-long
+        analytics_section = '''<li class="nav-item"><button type="button" class="btn-link" data-section="data_download">Data Download</button></li>'''  # pylint: disable=line-too-long
```

Comment:
I just noticed this was in there as well and was looking at it.

@jlajoie was this part of the bokchoy updates?